pkg/profiler/cpu/cpu.go

```diff
@@ -158,6 +159,8 @@ func NewCPUProfiler(
 
 		memlockRlimit: memlockRlimit,
 
+		processErrorTracker: map[error]int{},
```

Comment:
the `error` type can behave a bit weird here, I think we should just use `string` and use `err.Error()`